frontend/exporter/src/constant_utils/uneval.rs

```diff
@@ -111,8 +111,8 @@ pub fn translate_constant_reference<'tcx>(
         .try_normalize_erasing_regions(typing_env, ty)
         .unwrap_or(ty);
     let kind = if let Some(assoc) = s.base().tcx.opt_associated_item(ucv.def) {
-        if assoc.trait_item_def_id.is_some() {
```

Comment:
This condition is only true for trait impls, so we were missing default trait constants (that are children of the trait decl).